lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -514,6 +514,13 @@ function dosomething_global_get_language($account, stdClass $node = NULL, $respe
     // Respect the URL first.
     $request_path = explode('/', request_path());
     if (!empty($request_path[0])) {
+
+      // If the string is longer than a country code
```

Comment:
we should be able to follow the way we determine language in [other places](https://github.com/DoSomething/phoenix/blob/bd9b5fc7a3a585481cbff4a1435a7804bdd8f14e/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc#L93) instead of updating this